python/py-country/Portfile

```diff
@@ -58,6 +58,6 @@ if {${name} ne ${subport}} {
     }
 
     livecheck.type      none
+} else {
+    livecheck.type      pypi
```

Comment:
this is the default set by the `python` PG, so the `else` statement is redundant and you can remove it